src/pages/Home.js

```diff
@@ -16,7 +16,7 @@ export default function Home() {
             doutorado.
           </p>
 
-          <Link className="info-link" to="/">
+          <Link className="info-link" to="/Details">
```

Comment:
url é SEMPRE minúsculo
```suggestion
          <Link className="info-link" to="/details">
```